app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
+
+        //encode it as a zip file
+        var content = zip.encode(this._io, temp_dir);
+        this._io.writeFile(false, instanceName, new Buffer(content, 'base64'));
+
+        //delete the temp dir
+        function rmDirRecursive(io, dir){
+            function _isDirName(name) {
+                return name.slice(-1) === '/';
+            }
+
+            var i;
+            var entries = io.readDir(false, dir);
+            for (i in entries){
+                var entry = entries[i];
+                var fullPath = [dir, entry].join(dir.slice(-1) === '/' ? '' : '/');
```

Comment:
also about multi line `var` declarations, we have this piece of styleguide: https://github.com/metapolator/metapolator/wiki/Coding-style-and-substance#variable-declaration